api/config/default.json

```diff
@@ -38,5 +38,11 @@
   },
   "geolocation": {
     "gmapsApiKey": null
+  },
+  "password": {
```

Comment:
add these to the custom-environment-variables.json file too, so they can more easily be configured